include/ppx/grfx/grfx_pipeline.h

```diff
@@ -153,6 +153,12 @@ struct OutputState
     grfx::Format depthStencilFormat                          = grfx::FORMAT_UNDEFINED;
 };
 
+struct MultiViewState
```

Comment:
Could you define this somewhere in a common header so we can use it for grfx_pipeline and grfx_renderpass? (maybe here include/ppx/grfx/grfx_helper.h)?